cpp/src/command_classes/CommandClass.cpp

```diff
@@ -329,12 +329,15 @@ void CommandClass::ReadValueRefreshXML
 	char const* str;
 	bool ok = false;
 	const char *genre;
+	int32 intVal;
```

Comment:
why did you do this? I don't see any warnings on VS2015 and smells like a broken compiler rather than a problem.